dfanalyzer/graph_visualization/cytoscape.py

```diff
@@ -0,0 +1,352 @@
+import json
+import networkx as nx
+import ipycytoscape
+import ipywidgets as widgets
+from IPython.display import display
+# import ipycytoscape
+# import networkx as nx
+
+
+class CytoGraph:
+    
+    def get_json(self, graph, degree_dict = {}):
+        '''
+        Input: networkx graph,  dict (key = node_id and value = degree)
+        Returns: json_data (format needed for ipycytoscape visualization)
+        '''
+        # for pid_tid -> file bipartite graphs
+        json_data = {
+                    'nodes': [{'data': {'id': str(node[0]), 'degree': int(degree_dict[node[0]])}} for node in graph.nodes(data=True)],
+                    'edges': [{'data': {'source': str(edge[0]), 'target': str(edge[1]), 'directed': True}} for edge in graph.edges(data=True)]}
+
+        return json_data
+    
+    
+    def get_rich_json(self, graph, degree_dict = {}):
+        '''
+        Get json with Nodes and Edges attributes
+        Input: networkx graph,  dict (key = node_id and value = degree)
+        Returns: json_data (format needed for ipycytoscape visualization)
+        '''
+        # for pid_tid -> file bipartite graphs
```

Comment:
Do we need this commented lines if so. add a comment above why. if not remove them.